assignment-section-05/classification_model/pipeline.py

```diff
@@ -0,0 +1,73 @@
+# to persist the model and the scaler
```

Comment:
this comment? probably legacy. Pls remove

Reply:
removed